pkg/storage/kube/store_test.go

```diff
@@ -127,8 +127,10 @@ var (
 
 	// stubBuild is a build
 	stubBuild = &brigade.Build{
-		ID:        stubBuildID,
-		ProjectID: stubProjectID,
+		ID:         stubBuildID,
+		ProjectID:  stubProjectID,
+		ShortTitle: "this is a short title",
+		LongTitle:  "this is a long title",
```

Comment:
But the short title is longer than the long title! 😜

Reply:
> Is the vision for the Kashti dashboard to show the short title for each entry on the build list and then the long title (perhaps in addition to the short) on each individual build page?

That is exactly what I had in mind. Of course, I am open to feedback on that (or in this PR) if that doesn't jive with other maintainers.

Reply:
This makes sense - the one thing we should make sure in `brig` and Kashti is to fallback to displaying the build ID if the title is not available.